lints/lint_mp_modulus_must_be_2048_bits_or_more.go

```diff
@@ -0,0 +1,66 @@
+/*
+ * ZLint Copyright 2019 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/********************************************************************
+Section 5.1 - Algorithms
+RSA keys whose modulus size in bits is divisible by 8, and is at least 2048.
+********************************************************************/
+
+package lints
+
+import (
+	"crypto/rsa"
+
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/util"
+)
+
+type modulus2048OrMore struct{}
+
+func (l *modulus2048OrMore) Initialize() error {
+	return nil
+}
+
+func (l *modulus2048OrMore) CheckApplies(c *x509.Certificate) bool {
+	if c.PublicKeyAlgorithm == x509.RSA {
+		return true
+	}
+
+	return false
+}
+
+func (l *modulus2048OrMore) Execute(c *x509.Certificate) *LintResult {
+	pubKey, ok := c.PublicKey.(*rsa.PublicKey)
+	if !ok {
+		return &LintResult{Status: Fatal, Details: "certificate public key was not an RSA public key"}
+	}
+
+	bitLen := pubKey.N.BitLen()
+	if bitLen < 2048 {
+		return &LintResult{Status: Error}
+	}
+
+	return &LintResult{Status: Pass}
+}
+
+func init() {
+	RegisterLint(&Lint{
+		Name:          "e_mp_modulus_must_be_2048_bits_or_more",
+		Description:   "RSA keys whose modulus size in bits is divisible by 8, is at least 2048 and public exponent is not equal to 1",
```

Comment:
```suggestion
		Description:   "RSA keys must have modulus size of at least 2048 bits",
```

The != 1 and divisible by 8 portions of this description are now separate lints.